### PR TITLE
2nd attempt

### LINKED ISSUE #14: Remove cruft files
Current output of copier is shown below on the left side:

image

Cruft files or directories to remove:

* docs/css
* docs/reference
* unnecessary, duplicate directory of <listener-dummy>
* scripts/

### PR DESCRIPTION
**Pull request by kefeimo**
* Handling issue #14 
* To test: at the terminal, run `copier git@github.com:kefeimo/copier-poetry-volttron-agent.git output/ -r b4197c3` (assuming output copier-generated package to ./output/ folder

### CODE REVIEW

**Comment by craigpnnl** [comment]
@kefeimo I like the structure...though we should remove the cli.py and __main__ from the module....

remove the test or modify it so it tests the agent code not the cli

**Comment by craigpnnl** [comment]
@kefeimo I like the structure...though we should remove the cli.py and __main__ from the module....

**Comment by kefeimo** [comment]
Update
image

test with `copier git@github.com:kefeimo/copier-poetry-volttron-agent.git output/ -r 0bdcd06`

File: project/[[ python_package_distribution_name ]]/__init__.py.jinja [hunk 1]
```diff
@@ -7,4 +7,4 @@
 
 from typing import List
 
-__all__: List[str] = []  # noqa: WPS410 (the only __variable__ we use)
+__all__: List[str] = []  # noqa: WPS410 (the only __variable__ we use)
```
Comment (craigpnnl) [inline project/[[ python_package_distribution_name ]]/__init__.py.jinja]: If we are going to leave the __init__.py then it should have a new line here.

Comment (kefeimo) [reply project/[[ python_package_distribution_name ]]/__init__.py.jinja]: Updated. use the following command to test
`
copier git@github.com:kefeimo/copier-poetry-volttron-agent.git output/ -r b692591
`

File: project/scripts/post_generation.py.jinja [hunk 2]
```diff
@@ -46,4 +46,3 @@ if __name__ == "__main__":
 
     print("Project sucessfully generated!")
     print("Run `make` to show the available actions.")
-
```
Comment (craigpnnl) [inline project/scripts/post_generation.py.jinja]: line at the end

File: project/tests/test_cli.py.jinja [hunk 3]
```diff
@@ -20,4 +20,4 @@ def test_show_help(capsys):
     with pytest.raises(SystemExit):
         cli.main(["-h"])
     captured = capsys.readouterr()
-    assert "[[ python_package_command_line_name ]]" in captured.out
+    assert "[[ python_package_command_line_name ]]" in captured.out
```
Comment (craigpnnl) [inline project/tests/test_cli.py.jinja]: last line.

File: project/scripts/post_generation.py.jinja (hunk 2 above)
Comment (kefeimo) [reply project/scripts/post_generation.py.jinja]: Resolved

File: project/tests/test_cli.py.jinja (hunk 3 above)
Comment (kefeimo) [reply project/tests/test_cli.py.jinja]: Resolved

**Pull request merged**